src/igz_controls/components/default-dropdown/default-dropdown.component.js

```diff
@@ -43,6 +43,7 @@
     angular.module('iguazio.dashboard-controls')
         .component('igzDefaultDropdown', {
             bindings: {
+                additionalClass: '@',
```

Comment:
I think this binding should be optional:
```suggestion
                additionalClass: '@?',
```